test/test_trmm.cc

```diff
@@ -86,8 +85,8 @@ void test_trmm_work( Params& params, bool run )
         printf( "\n"
                 "A Am=%5lld, Am=%5lld, lda=%5lld, size=%10lld, norm=%.2e\n"
                 "B Bm=%5lld, Bn=%5lld, ldb=%5lld, size=%10lld, norm=%.2e\n",
-                (lld) Am, (lld) Am, (lld) lda, (lld) size_A, Anorm,
-                (lld) Bm, (lld) Bn, (lld) ldb, (lld) size_B, Bnorm );
+                llong( Am ), llong( Am ), llong( lda ), llong( size_A ), Anorm,
```

Comment:
We have twice `Am`. Is it normal?

Reply:
I think it was just to line up with the rectangular B matrix. A is an m-by-m matrix, B is m-by-n.